engine/src/test/java/org/hibernate/validator/test/internal/engine/typeannotationconstraint/TypeAnnotationConstraintTest.java

```diff
@@ -250,48 +646,38 @@ public void unsupported_use_of_type_constraints_logs_warning() {
 		log4jRootLogger.removeAppender( assertingLogger );
 	}
 
-	@Test
-	@TestForIssue(jiraKey = "HV-1121")
-	public void property_path_contains_index_information() {
-		A1 a = new A1();
-		a.names = Arrays.asList( "" );
-
-		Set<ConstraintViolation<A1>> constraintViolations = validator.validate( a );
-
-		assertNumberOfViolations( constraintViolations, 1 );
-
-		Iterator<Path.Node> propertyPathIterator = constraintViolations.iterator().next().getPropertyPath().iterator();
-
-		Path.Node firstNode = propertyPathIterator.next();
-		assertThat( firstNode.getIndex() ).isNull();
-		assertThat( firstNode.getName() ).isEqualTo( "names" );
-		assertThat( firstNode.getKind() ).isEqualTo( ElementKind.PROPERTY );
-
-		Path.Node secondNode = propertyPathIterator.next();
-		assertThat( secondNode.getIndex() ).isEqualTo( 0 );
-		assertThat( secondNode.getName() ).isEqualTo( "<collection element>" );
-		assertThat( secondNode.getKind() ).isEqualTo( ElementKind.PROPERTY );
+	@SafeVarargs
+	private static <T> Set<T> newSet(T... elements) {
+		Set<T> set = new HashSet<T>();
+		for ( T element : elements ) {
+			set.add( element );
+		}
+		return set;
 	}
 
-	static class A1 {
+	// List
+
+	static class List1 {
 		@Valid
 		List<@NotNull @NotBlank String> names;
 	}
 
-	static class A2 {
+	static class List2 {
```

Comment:
Maybe `TypeWithList2`? When looking at the usage above, I first thought the type itself would be a list type.